openpype/hosts/houdini/hooks/set_default_display_and_view.py

```diff
@@ -0,0 +1,53 @@
+from openpype.lib.applications import PreLaunchHook, LaunchTypes
+from openpype.settings import get_project_settings
+
+
+class SetDefaultDislayView(PreLaunchHook):
+    """Set default view and default display for houdini host that use OpenColorIO.
+
+    Houdini's defaultDisplay and defaultView are set by
+    setting 'OCIO_ACTIVE_DISPLAYS' and 'OCIO_ACTIVE_VIEWS'
+    environment variables respectively.
+
+    More info: https://www.sidefx.com/docs/houdini/io/ocio.html#set-up
+    """
+
+    app_groups = {"houdini"}
+    launch_types = {LaunchTypes.local}
+
+    def execute(self):
+
+        OCIO = self.launch_context.env.get("OCIO")
+
+        # This is a cheap way to skip this hook if either
+        # global color management or houdini color management was disabled.
```

Comment:
```suggestion
        # This is a cheap way to skip this hook if either global color 
        # management or houdini color management was disabled because the
        # OCIO var would be set by the global OCIOEnvHook
```